app/src/main/java/org/literacyapp/authentication/TrainingHelper.java

```diff
@@ -314,15 +314,29 @@ private File createAvatarFileFromStudentImage(StudentImage studentImage, Student
 
     /**
      * Find similar students
-     * Case 1: Student was added during fallback but in the meantime the same person has an existing StudentImageCollectionEvent and a new Student entry
+     *
+     *
      */
     public synchronized void findAndMergeSimilarStudents(){
         Log.i(getClass().getName(), "findAndMergeSimilarStudents");
         PreProcessorFactory ppF = new PreProcessorFactory(context);
+        TensorFlow tensorFlow = getInitializedTensorFlow();
+        findSimilarStudentsUsingAvatarImages(ppF, tensorFlow);
```

Comment:
Does an avatar image also include those taken manually in the fall-back registration activity? If so, you currently get a mix between 180px and 224px images. Will this inconsistency in image size/format cause problems?